kmod/src/file.c

```diff
@@ -111,7 +112,7 @@ ssize_t scoutfs_file_aio_write(struct kiocb *iocb, const struct iovec *iov,
 	if (ret)
 		goto out;
 
-	if (scoutfs_per_task_add_excl(&si->pt_data_lock, &pt_ent, inode_lock)) {
+	if (scoutfs_per_task_add_excl(&si->pt_data_lock, &pt_ent, scoutfs_inode_lock)) {
```

Comment:
This confused me for a while.  This hunk should be down in the i_mutex->i_rwsem patch where the lock was originally renamed.